eng/common/scripts/copy-docs-to-blobstorage.ps1

```diff
@@ -1,15 +1,27 @@
 # Note, due to how `Expand-Archive` is leveraged in this script,
 # powershell core is a requirement for successful execution.
 param (
+  # used by VerifyPackages
```

Comment:
Why did you add all these parameters to the front of the list?

Reply:
Does the order matter? All these parameters execute in the beginning.

Reply:
Thinking about this a little this is a log of extra parameters just for the one step below. Perhaps we should write a simple wrapper script or function in the artifacts-metadata-parsing.ps1 that can do all this work for us and simply set the release tag and we only pass the release tag to this script.

Reply:
I evaluated all params for verifyPackages. What we need is only the release tag, so the `releaseSha` is the only one we need for copy-blob script. It is already a function in `artifact-metadata-parsing` script, so extract to another function seems redundant.